pyNN/utility/__init__.py

```diff
@@ -374,14 +374,15 @@ def __call__(self, t):
         return t + self.interval
 
 
+#deprecated
```

Comment:
If these functions are no longer used, you can delete them completely rather than just marking as deprecated.